Healthy/Classes/Generated/UIImage.Generated.swift

```diff
@@ -9,4 +9,12 @@ extension UIImage {
     static var accentColor: UIImage {
       UIImage(named: "AccentColor")!
     }
+
+    static var backgroundSplash: UIImage {
+      UIImage(named: "background-splash")!
+    }
+
+    static var iconSplash: UIImage {
+      UIImage(named: "icon-splash")!
```

Comment:
```suggestion
      UIImage(named: "background-splash")!
    }

    static var iconSplash: UIImage {
      UIImage(named: "icon-splash")!
```

This is the naming convention we need to follow 
icon-...
background-...

Reply:
I did not see any change!
I use icon-...
background-...

Reply:
@ahmdmhasn, thank you for your revision😍🤝